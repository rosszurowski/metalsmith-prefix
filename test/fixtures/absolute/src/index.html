<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
	<a href="/absolute/link">An absolute link</a>
</body>
</html>