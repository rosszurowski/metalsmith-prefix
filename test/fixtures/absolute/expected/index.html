<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
	<a href="/prefix/absolute/link">An absolute link</a>
</body>
</html>